agent/base.ut.pre

```diff
@@ -10,3 +10,6 @@ fi
 if [[ -z "${_pbench_full_hostname}" ]]; then
     export _pbench_full_hostname="testhost.example.com"
 fi
+if [[ -z "${_pbench_hostname_ip}" ]]; then
+    export _pbench_hostname_ip="42.42.42.42"
```

Comment:
While that's cute, I wonder if something that's not routed like 172.0.0.0 might be "safer"?

Reply:
Fixed.